test/e2e/revision_test.go

```diff
@@ -76,18 +93,34 @@ func (test *e2eTest) revisionListWithService(t *testing.T, serviceNames ...strin
 	}
 }
 
-func (test *e2eTest) revisionDelete(t *testing.T, serviceName string) {
-	revName := test.findRevision(t, serviceName)
-
+func (test *e2eTest) revisionDelete(t *testing.T, revName string) {
 	out, err := test.kn.RunWithOpts([]string{"revision", "delete", revName}, runOpts{})
 	assert.NilError(t, err)
 
 	assert.Check(t, util.ContainsAll(out, "Revision", revName, "deleted", "namespace", test.kn.namespace))
 }
 
-func (test *e2eTest) revisionDescribeWithPrintFlags(t *testing.T, serviceName string) {
-	revName := test.findRevision(t, serviceName)
+func (test *e2eTest) revisionMultipleDelete(t *testing.T, revisionNames []string) {
+	existRevision1 := revisionNames[0]
+	existRevision2 := revisionNames[1]
+	nonexistRevision := revisionNames[2]
+	out, err := test.kn.RunWithOpts([]string{"revision", "list"}, runOpts{NoNamespace: false})
+	assert.NilError(t, err)
+	assert.Check(t, strings.Contains(out, existRevision1), "Required revision1 does not exist")
+	assert.Check(t, strings.Contains(out, existRevision2), "Required revision2 does not exist")
+	assert.Check(t, !strings.Contains(out, nonexistRevision), "Nonexistent revision does exist")
+
+	out, err = test.kn.RunWithOpts([]string{"revision", "delete", existRevision1, existRevision2, nonexistRevision}, runOpts{NoNamespace: false})
+
+	expectedSuccess1 := fmt.Sprintf(`Revision '%s' successfully deleted in namespace '%s'`, existRevision1, test.kn.namespace)
+	expectedSuccess2 := fmt.Sprintf(`Revision '%s' successfully deleted in namespace '%s'`, existRevision2, test.kn.namespace)
+	expectedErr := fmt.Sprintf(`revisions.serving.knative.dev "%s" not found.`, nonexistRevision)
+	assert.Check(t, strings.Contains(out, expectedSuccess1), "Failed to get 'successfully deleted' first revision message")
```

Comment:
```suggestion
	assert.Check(t, util.ContainsAll(out, "Revision", revisionNames[0], "deleted", "namespace", test.kn.namespace))
```

Reply:
comparing the exact string is fine but if you see other tests, we only check for certain keywords in the error message using `util.ContainsAll`

Reply:
I've made the change, but because util.ContainsAll isn't limiting it by line it means if say existRevision1 and nonexistRevision are switched the test still passes.